pkg/reactable/bloom.go

```diff
@@ -133,12 +143,16 @@ func (s *DendroCore) Attack(atk *combat.AttackEvent, evt glog.Event) (float64, b
 		x, y := s.Gadget.Pos()
 		enemies := s.Core.Combat.EnemyByDistance(x, y, combat.InvalidTargetKey)
 		if len(enemies) > 0 {
-			s.Core.QueueAttack(ai, combat.NewCircleHit(s.Core.Combat.Enemy(enemies[0]), 1, false, combat.TargettableEnemy), -1, 5)
+			ap := combat.NewCircleHit(s.Core.Combat.Enemy(enemies[0]), 1)
+			s.Core.QueueAttack(ai, combat.NewCircleHit(s.Core.Combat.Enemy(enemies[0]), 1), -1, 5)
```

Comment:
The `ap` variable is created here despite not being used until later. I assume it's supposed to be used in this line?